subsys/caf/modules/leds.c

```diff
@@ -32,7 +32,15 @@ struct led {
 	struct k_delayed_work work;
 };
 
-#define DT_DRV_COMPAT pwm_leds
+#ifdef CONFIG_CAF_LEDS_PWM
+  #define DT_DRV_COMPAT pwm_leds
+  #define DEFAULT_LABEL(id) "LED_PWM_"#id
```

Comment:
USE _CONCAT or similar macro. There would be problem with double expansion.
`#define DEFAULT_LABEL(id) STRINGIFY(_CONCAT(LED_PWM_, id))`

Reply:
I can see that you are using this macro in other macros - but just to be on the safe side.